src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/Configuration.php

```diff
@@ -83,6 +94,25 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end();
 
+        $this->addDebugSection($root);
```

Comment:
Any specific reason you have moved that into an own method? If no I would not do it (for the sake of consistency).